### DISCUSSION TITLE
USB-C

### DISCUSSION THREAD

**Discussion by watsterito** [mwood77/winderoo#15] [General]
Is there an ESP32 with USB-C that will work?
I'd rather not use Micro USB.

Thanks!

**Comment by mwood77** [comment]
Sure. You could try one of these, but I'm not very familiar with them:
- [ESP32 Type C](https://s.click.aliexpress.com/e/_DByIg8l)
- [ESP32 S3](https://nl.aliexpress.com/item/1005006266375800.html?gatewayAdapt=glo2nld)

**Reply by watsterito** [reply]
This worked fine.
